src/app/components/elements/CareersBanner.tsx

```diff
@@ -19,50 +21,92 @@ const CsAtWorkDescription = () => {
   return <p className="mb-3">{!user?.loggedIn ? loggedOutDescription : roleSpecificDescription}</p>;
 };
 
+interface CareerCardProps {
+  imgSrc?: string;
+  imgAlt?: string;
+  title: string;
+  text: string | JSX.Element;
+  linkTo: string;
+  linkText: string;
+  children?: React.ReactNode;
+}
+
+const CareerCard = ({ imgSrc, imgAlt, title, text, linkTo, linkText, children }: CareerCardProps) => (
+  <Card className="career-card">
+    {imgSrc && <CardImg variant="top" src={imgSrc} alt={imgAlt} className="career-media-row-image" />}
+    <CardBody className="career-card-body">
+      {children}
+      <CardTitle className="career-subtitle">{title}</CardTitle>
+      <CardText className="career-text">{text}</CardText>
+      <div className="career-link-column">
+        <Link className="career-link" to={linkTo}>
+          {linkText}
+        </Link>
+      </div>
+    </CardBody>
+  </Card>
+);
+
+CareerCard.propTypes = {
```

Comment:
Small fyi, PropTypes is deprecated and is mostly redundant in TS apps, so don't worry about spending time setting these up. Unless there's a specific functional need for them in this application?